uitools/import/Esri/ArcGISRuntime/Toolkit/BookmarksView.qml

```diff
@@ -0,0 +1,90 @@
+/*******************************************************************************
+ *  Copyright 2012-2022 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ ******************************************************************************/
+import Esri.ArcGISRuntime.Toolkit.Controller 100.15
+
+import QtQuick 2.15
+import QtQml 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+
+/*!
+ \qmltype BookmarksView
+ \inqmlmodule Esri.ArcGISRuntime.Toolkit
+ \ingroup ArcGISQtToolkitUiQmlViews
+ \since Esri.ArcGISRuntime 100.15
+ \brief The user interface for the BookmarksView.
+
+ The BookmarksView displays a collection of viewpoints represented by bookmarks from either ArcGIS Online, a user-defined portal,
+ or an array of Bookmarks. When the user selects a bookmark from the provied list, the currently rendered bookmark in the
+ geoView is replaced with the new bookmark view extent.
```

Comment:
❌  Copy+paste error here from BasemapGallery. The `Bookmarks` are associated with the `Geomodel`, so at most they can come from a Webmap/Webscene or be programmatically defined.